fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs

```diff
@@ -133,7 +133,12 @@ pub fn main() {
     let scheduler = QueueScheduler::new();
 
     // A fuzzer with feedbacks and a corpus scheduler
+    #[cfg(not(feature = "bloom_filter"))]
     let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
+    #[cfg(feature = "bloom_filter")]
+    let mut fuzzer =
+        StdFuzzer::new_with_bloom_filter(scheduler, feedback, objective, 10_000_000, 0.001)
```

Comment:
- no `new_` here, we never have that on complex constructors
- Instead of `with_bloom_filter`, name it according to the feature (`with_reexecution_filter`/`with_bloom_input_filter`/?)